src/rf/js/src/core/models.js

```diff
@@ -61,15 +79,15 @@ var Layer = Backbone.Model.extend({
     },
 
     isPreValidated: function() {
-        return this.get('status') === STATUS_CREATED;
+        return this.get('status_created');
```

Comment:
pre-validated means the layer has not reached the stage of validation. But status_created will be true at any point after creation. I think the name of this method should be changed to isCreated.